vocdoni/bridge-ui
# feature/vot-171-change-logo-to-aragon-voice-favicon

# Short description
Changes:
- app title
- header logo
- favicon
- apple touch icon added
- safari pinned tab icon added
- android chrome icon added
- browser config xml added with mstile icon

# How can it be tested
Open the home page to see the app title, favicon and header logo. Pin the tab on safari to see the pinned tab icon. Bookmark the page in chrome or add as a top site in firefox to see the chrome icon, pin as a tile in windows to see mstile

# Tracker ID
VOT-171
